tendrl/gluster_integration/sds_sync/vol_utilization.py

```diff
@@ -41,15 +44,17 @@ def showVolumeUtilization(vnames):
             used_inode = data.f_files - data.f_ffree
             total_inode = data.f_files
             pcnt_inode_used = (float(used_inode) / total_inode) * 100
-            ret_dict[volume] = {
-                'total': total_size,
-                'free': free_size,
-                'used': used_size,
-                'pcnt_used': vol_utilization,
-                'total_inode': total_inode,
-                'used_inode': used_inode,
-                'pcnt_inode_used': pcnt_inode_used
+            success = { volume : {
```

Comment:
Same as for `error` I would except `ret_det` to be a dict with two keys `error` and `success` and their values should be dicts with volume names as keys and their values would be utilization details in case of success and error message in case of error.

Reply:
this is what is happening.
error dict created at line#30 and success dict created at line#47 are assigned to the error and success list respectively in the ret_dict.